### DISCUSSION TITLE
telegram群进不去.大佬.通过下哈

### DISCUSSION THREAD

**Discussion by xucongyong** [bmax121/APatch#432] [Q&A] [answered]
telegram群进不去.大佬.通过下哈.
我的ID是HPrince

**Comment by pomelohan** [comment, marked as answer]
中文组不接受动手能力差、没有基础知识的小白，请仔细学习后做题加入。如果自己无法完成做题，可以考虑加入 ZH/EN 的公共群聊，不需要做题申请，即刻申请即刻加入

**Reply by xucongyong** [reply]
[
![Screenshot 2024-05-12 at 08 10 02](https://github.com/bmax121/APatch/assets/22570151/a7ecfc1d-b3a4-4f5d-99cd-a7e561f60155)
](url)

**Reply by xucongyong** [reply]
the admins of this group have restricted your ability to send  messages

**Reply by pomelohan** [reply]
1、要么是你违反了群规
2、你没有通过 Rose 的人机验证

**Reply by pomelohan** [reply]
看了一下你的账号，没有通过 Rose 的人机验证，可以尝试退群重新进入一次然后点一下人机验证
